Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -286,7 +291,7 @@ file that was distributed with this source code.
         {% endif %}
 
         <div class="{{ div_class }}">
-            {{ form_widget(form, {'horizontal': false, 'horizontal_input_wrapper_class': ''}) }} {# {'horizontal': false, 'horizontal_input_wrapper_class': ''} needed to avoid MopaBootstrapBundle messing with the DOM #}
```

Comment:
shall we really kill the comment here?, maybe we can set this above the code line to avoid to repeat the code itself?